src/anyp/Uri.cc

```diff
@@ -377,8 +391,8 @@ AnyP::Uri::parse(const HttpRequestMethod& method, const char *url)
     }
 
     /* For IPV6 addresses also check for a colon */
```

Comment:
This comment should be moved into urlAppendDomain(), right? It seems to be about that `strchr(:)` check there. I think it is a good idea to keep it because it is not obvious why the code checks for a colon.

In fact, it is also not obvious that the code checks for a period to, in part, exclude IPv4 addresses! Your call whether to clarify that with another comment.

Reply:
done